### PR TITLE
Update ClickHouse sink

### LINKED ISSUE #2681: Bug: clickhouse doc behind.
### Where's the bug?

https://docs.risingwave.com/docs/current/sink-to-clickhouse/

### Describe the bug

We support creating upsert sinks for CollapsingMergeTree and VersionedCollapsingMergeTree engines. RisingWave will transform DELETE into INSERT SIGN = 1.

=> 

We support creating upsert sinks for CollapsingMergeTree and VersionedCollapsingMergeTree engines. RisingWave will transform DELETE into INSERT SIGN = -1.
We support creating upsert sinks for ReplacingMergeTree  engines. RisingWave will transform DELETE into INSERT SIGN = 1.

Parameters add :
clickhouse.delete.column: Optional You need to add this parameter at the same time when you use ck's ReplacingMergeTree  and set up the delete column, you can run upsert sink under the ReplacingMergeTree engine

### PR DESCRIPTION
**Pull request by WanYixian**
## Description

Update supported engine and add `clickhouse.delete.column` parameter

## Related doc issue

Resolve https://github.com/risingwavelabs/risingwave-docs/issues/2681

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/sink-to-clickhouse.md [hunk 1]
```diff
@@ -47,12 +47,15 @@ WITH (
...
+| `clickhouse.delete.column` | Optional. Add this parameter when using ClickHouse's ReplacingMergeTree and setting up the delete column. You can run an upsert sink using the ReplacingMergeTree engine.|
 
 ### Upsert sinks
 
 While RisingWave supports `append-only` sinks for all ClickHouse engines, support for `upsert` sinks is limited. Additionally, for ReplacingMergeTree engines, an `append-only` sink will not insert duplicate data.
 
-We support creating `upsert` sinks for CollapsingMergeTree and VersionedCollapsingMergeTree engines. RisingWave will transform `DELETE` into `INSERT SIGN = 1`.
+We support creating `upsert` sinks for CollapsingMergeTree and VersionedCollapsingMergeTree engines. RisingWave will transform `DELETE` into `INSERT SIGN = -1`.
```
Comment (hengm3467) [inline docs/guides/sink-to-clickhouse.md]: I feel the wording of these two paragraphs can be improved. One suggested version below. You can improve the styling of the content with proper use of lists and higlighting. 

RisingWave supports 'upsert' sinks for the following ClickHouse engines:

- CollapsingMergeTree and VersionedCollapsingMergeTree: DELETE operations are transformed into INSERT with SIGN = -1
- ReplacingMergeTree: DELETE operations are transformed into INSERT with SIGN = 1

Comment (WanYixian) [reply docs/guides/sink-to-clickhouse.md]: Yes this looks more organized. Updated.

File: docs/guides/sink-to-clickhouse.md [hunk 2]
```diff
@@ -47,12 +47,17 @@ WITH (
...
 | `commit_checkpoint_interval`| Optional. Commit every N checkpoints (N > 0). Default value is 10. <br/>The behavior of this field also depends on the `sink_decouple` setting:<ul><li>If `sink_decouple` is true (the default), the default value of `commit_checkpoint_interval` is 10.</li> <li>If `sink_decouple` is set to false, the default value of `commit_checkpoint_interval` is 1.</li> <li>If `sink_decouple` is set to false and `commit_checkpoint_interval` is set to larger than 1, an error will occur.</li></ul>|
+| `clickhouse.delete.column` | Optional. Add this parameter when using ClickHouse's ReplacingMergeTree and setting up the delete column. You can run an upsert sink using the ReplacingMergeTree engine.|
 
 ### Upsert sinks
 
 While RisingWave supports `append-only` sinks for all ClickHouse engines, support for `upsert` sinks is limited. Additionally, for ReplacingMergeTree engines, an `append-only` sink will not insert duplicate data.
 
-We support creating `upsert` sinks for CollapsingMergeTree and VersionedCollapsingMergeTree engines. RisingWave will transform `DELETE` into `INSERT SIGN = 1`.
+RisingWave supports `upsert` sinks for the following ClickHouse engines:
```
Comment (hengm3467) [inline docs/guides/sink-to-clickhouse.md:56]: Better than my version. Awesome! Thank you.

File: docs/guides/sink-to-clickhouse.md [hunk 3]
```diff
@@ -47,12 +47,17 @@ WITH (
 | `clickhouse.database`  | Required. Name of the ClickHouse database that you want to sink data to.|
 | `clickhouse.table`      | Required. Name of the ClickHouse table that you want to sink data to.|
 | `commit_checkpoint_interval`| Optional. Commit every N checkpoints (N > 0). Default value is 10. <br/>The behavior of this field also depends on the `sink_decouple` setting:<ul><li>If `sink_decouple` is true (the default), the default value of `commit_checkpoint_interval` is 10.</li> <li>If `sink_decouple` is set to false, the default value of `commit_checkpoint_interval` is 1.</li> <li>If `sink_decouple` is set to false and `commit_checkpoint_interval` is set to larger than 1, an error will occur.</li></ul>|
+| `clickhouse.delete.column` | Optional. Add this parameter when using ClickHouse's ReplacingMergeTree and setting up the delete column. You can run an upsert sink using the ReplacingMergeTree engine.|
```
Comment (hengm3467) [inline docs/guides/sink-to-clickhouse.md]:
```suggestion
| `clickhouse.delete.column` | Optional. You can run an upsert sink using the ReplacingMergeTree engine. When using the ReplacingMergeTree engine, you can specify the delete column with this parameter. |
```

Comment (hengm3467) [reply docs/guides/sink-to-clickhouse.md]: Not sure if my rewording is accurate. @xxhZs Please review. Thanks.

**Review by xxhZs — APPROVED** [review]

**Review by hengm3467 — APPROVED** [review]

**Pull request merged**